cyraft/node.py

```diff
@@ -88,6 +88,8 @@ def __init__(self) -> None:
 
         ## Volatile state on leaders
         self._next_index: typing.List[int] = []  # index of the next log entry to send to that server
+
+
         # self._match_index: typing.List[int] = []  # index of highest log entry known to be replicated on server
```

Comment:
Actually now that I think of it: we're always using Black formatter for all of the code.

In order to avoid these formatting type of comments/discussions.

See pycyphal:

<img width="637" alt="image" src="https://github.com/OpenCyphal-Garage/cyraft/assets/39975120/9a717f38-d0cc-421e-a3ce-4ce0a4882645">

(This is also something that should be part of CI, code that is not formatted correctly should fail)